app/assets/stylesheets/desktop/user-card.scss

```diff
@@ -1,317 +1,253 @@
-      background-position: center;
-      background-size: $size;
-      color: $primary;
-      .d-icon {
-        margin: auto;
-        font-size: $size / 1.5 !important;
-      }
-      &.rounded {
-        border-radius: 50%;
+      .location,
+      .website-name {
+        display: flex;
+        overflow: hidden;
+        align-items: center;
+        .d-icon {
+          margin-right: 0.25em;
+        }
       }
-    }
-  }
-
-  .members {
-    padding: 0 0 10px 0;
-    a {
-      color: lighten($primary, 40%);
-      &:hover {
+      .website-name a,
+      .location span {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
         color: $primary;
       }
+      .location {
+        margin-right: 0.5em;
+      }
+      .website-name a {
+        text-decoration: underline;
+      }
     }
   }
-
-  p {
-    margin: 0 0 5px 0;
-  }
-
-  .btn {
-    margin: 0 0 7px 0;
-    box-sizing: border-box;
-  }
-
-  .usercard-controls {
-    list-style-type: none;
-    float: right;
-    margin: 0;
-
-    button {
-      width: 100%;
-      min-width: 150px;
+  // custom user fields
+  .fifth-row {
+    .public-user-fields {
+      margin: 0;
     }
   }
-
-  .new-user a {
-    color: scale-color($user_card_background, $lightness: 70%);
-  }
-
-  .names {
-    float: left;
-    width: 45%;
-
-    span {
-      display: block;
-      width: 250px;
+  // badges
+  .sixth-row {
+    .badge-section {
+      display: flex;
+      align-items: flex-start;
+      .user-badge {
+        margin: 0 0.5em 0 0;
+        background: $primary-very-low;
+        border: 1px solid $primary-low;
+        color: $primary;
+      }
     }
   }
+}
 
-  .badge-section {
+// styles for group cards
+#group-card {
+  // avatar - names and controls
+  .first-row {
     display: flex;
-    align-items: flex-start;
-    width: 500px;
-    padding-bottom: 10px;
-    margin-top: 5px;
-
-    .user-badge {
-      margin-right: 0.5em;
-      background: $primary-very-low;
-      border: 1px solid $primary-low;
-      color: $user_card_primary;
-    }
-
-    h3 {
-      color: $user_card_background;
-      font-size: $font-0;
-      margin-bottom: -8px;
+    .group-card-avatar {
+      margin-top: $avatar_margin;
+    }
+    .group-card-avatar {
+      .avatar-flair {
+        width: $avatar_width;
+        height: $avatar_width;
+        display: flex;
+        color: $primary;
+        .d-icon {
+          margin: auto;
+          font-size: $avatar_width / 1.5;
+        }
+        &.rounded {
+          border-radius: 50%;
+        }
+      }
     }
   }
-
-  .more-user-badges {
-    @extend .user-badge;
-    padding: 3px 8px;
-  }
-
-  .suspended {
-    color: $danger;
-    margin-bottom: 5px;
-    clear: left;
-    padding-top: 15px;
+  // group bio
+  .second-row {
+    max-height: 150px;
+    overflow: auto;
+    .bio {
+      a {
+        color: $primary;
+        text-decoration: underline;
+      }
+      img {
```

Comment:
can we use class here please ?

Reply:
I've removed the rule for user cards because we swap images with `[image]` 

This is not the case for group cards. Since the image is something that the user inputs, we cannot use a class.